.orchestra/config/components/readline.yml

```diff
@@ -0,0 +1,42 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:overlay", "overlay")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+#@ load("/lib/make.lib.yml", "make")
+
+#@ readline_ver = "8.1.2"
+#@ source_url = "https://ftp.gnu.org/gnu/readline/readline-{}.tar.gz".format(readline_ver)
```

Comment:
We use `+`, sad but either convert them all (are fstrings supported?) or stick with old-school string concatenation.

Reply:
Starlark does not support f-strings. Using `+` is very sad though

Reply:
Use `+` or fix all other places too to use format strings.
Arguably anonymous `.format` is less readable than concatenation.